File: C3/C3 08 AS 03 Loops/05.cpp
//05. Write a C++ program to calculate sum of first N natural numbers.
#include<iostream>
using namespace std;
int main()
{
    int n, sum=0;
    cout<< "Enter a number: ";
    cin>>n;
    cout<< "Sum of first " << n << " natural number is: ";
    for(n;n;n--)
        sum=sum+n;
    cout<<sum;
    return 0;
}

File: Python/PY-25-AS-11-more-on-while/05.py
#   05. Write a python script to calculate sum of first N even natural numbers.
n = int(input("Enter a number: "));
print("Sum of first %d Even natural number is: "%(n),end="");
sum = 0;
while n:
    sum = sum + (2*n);
    n-=1;
print(sum);